dev/lsp/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
2,17],[43,17],[9,17],[8,17],[2,17],[10,17],[11,17],[12,17],[13,17],[14,17],[15,17],[16,17],[17,17],[3,17],[4,17],[21,17],[18,17],[19,17],[20,17],[22,17],[23,17],[24,17],[5,17],[25,17],[26,17],[27,17],[28,17],[6,17],[32,17],[29,17],[30,17],[31,17],[33,17],[7,17],[34,17],[39,17],[40,17],[35,17],[36,17],[37,17],[38,17],[1,17],[41,17]],"exportedModulesMap":[[107,1],[108,1],[110,2],[111,3],[112,4],[113,5],[114,6],[115,7],[116,8],[117,9],[118,10],[119,11],[120,11],[122,12],[121,13],[123,12],[124,14],[125,15],[109,16],[159,17],[126,18],[127,19],[128,20],[160,21],[129,22],[130,23],[131,24],[132,25],[133,26],[134,27],[135,28],[136,29],[137,30],[138,31],[139,31],[140,32],[141,33],[143,34],[142,35],[144,36],[145,37],[146,17],[147,38],[148,39],[149,40],[150,41],[151,42],[152,43],[153,44],[154,45],[155,46],[156,47],[157,48],[158,49],[168,50],[58,51],[50,52],[55,53],[48,17],[51,54],[49,55],[47,17],[57,56],[53,57],[46,17],[54,58],[52,59],[56,60],[162,61],[163,62],[45,17],[82,63],[81,64],[59,65],[69,66],[64,66],[63,67],[80,68],[66,66],[78,66],[73,67],[65,66],[60,66],[77,66],[76,66],[72,66],[74,69],[79,70],[68,67],[67,66],[70,66],[71,67],[61,66],[75,66],[62,67],[164,71],[165,72],[100,17],[44,17],[99,73],[86,74],[84,74],[94,74],[89,74],[93,74],[92,74],[90,74],[97,74],[96,75],[83,74],[87,74],[98,76],[88,74],[95,77],[91,74],[85,74],[161,17],[166,78],[167,79],[106,17],[103,80],[101,17],[102,81],[170,94],[104,95],[169,96],[105,97],[172,98],[197,99],[198,100],[173,101],[176,101],[195,99],[196,99],[186,99],[185,102],[183,99],[178,99],[191,99],[189,99],[193,99],[177,99],[190,99],[194,99],[179,99],[180,99],[192,99],[174,99],[181,99],[182,99],[184,99],[188,99],[199,103],[187,99],[175,99],[212,104],[211,98],[206,103],[208,105],[207,103],[200,103],[201,103],[203,103],[205,103],[209,105],[210,105],[202,105],[204,105],[213,98],[42,98],[43,98],[9,98],[8,98],[2,98],[10,98],[11,98],[12,98],[13,98],[14,98],[15,98],[16,98],[17,98],[3,98],[4,98],[21,98],[18,98],[19,98],[20,98],[22,98],[23,98],[24,98],[5,98],[25,98],[26,98],[27,98],[28,98],[6,98],[32,98],[29,98],[30,98],[31,98],[33,98],[7,98],[34,98],[39,98],[40,98],[35,98],[36,98],[37,98],[38,98],[1,98],[41,98]],"semanticDiagnosticsPerFile":[107,108,110,111,112,113,114,115,116,117,118,119,120,122,121,123,124,125,109,159,126,127,128,160,129,130,131,132,133,134,135,136,137,138,139,140,141,143,142,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,168,58,50,55,48,51,49,47,57,53,46,54,52,56,162,163,45,82,81,59,69,64,63,80,66,78,73,65,60,77,76,72,74,79,68,67,70,71,61,75,62,164,165,100,44,99,86,84,94,89,93,92,90,97,96,83,87,98,88,95,91,85,161,166,167,106,103,101,102,170,171,104,169,105,172,197,198,173,176,195,196,186,185,183,178,191,189,193,177,190,194,179,180,192,174,181,182,184,188,199,187,175,212,211,206,208,207,200,201,203,205,209,210,202,204,213,42,43,9,8,2,10,11,12,13,14,15,16,17,3,4,21,18,19,20,22,23,24,5,25,26,27,28,6,32,29,30,31,33,7,34,39,40,35,36,37,38,1,41],"latestChangedDtsFile":"./out/utils.d.ts"},"version":"5.0.4"}
```

Comment:
I am not familiar with what files are typically checked into a repo in TS.. this file seems very much generated.. can you double check for all files?